app/views/agency/irs/index.html.erb

```diff
@@ -17,13 +17,6 @@
     <% end %>
    </div>
   <% end %>
-  <div class="relative">
-    <%= image_tag 'irs/home.jpg' %>
-    <form action="/auth/saml" method="GET" class="absolute top-0 right-0 py2 px1">
-      <button type="submit" class="btn btn-outline bg-white navy">
-        <%= image_tag 'login-gov.svg', class: "align-middle", width: "125" %>
-      </button>
-    </form>
-  </div>
+  <%= render partial: 'shared/agency_form', locals: {image_url: 'irs/home.jpg'}  %>
```

Comment:
Please add a space before and after brackets. On a separate note, perhaps we should be using Slim across all repos for consistency?